workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/EntityGameObjectLinker.cs

```diff
@@ -76,6 +95,17 @@ public void LinkGameObjectToEntity(GameObject gameObject, Entity entity, ViewCom
             spatialOSComponent.SpatialEntityId = spatialEntityId;
         }
 
+        /// <summary>
+        ///     Un-links a GameObject and ECS Entity.
+        /// </summary>
+        /// <remarks>
+        ///    The GameObject and ECS Entity should already be linked before this method call.
+        /// </remarks>
+        /// <param name="gameObject">The GameObject to unlink.</param>
+        /// <param name="entity">The ECS Entity to unlink.</param>
+        /// <param name="viewCommandBuffer">
+        ///     An instance of the ViewCommandBuffer. Must be flushed after this method returns.
```

Comment:
Again it doesn't have to be done immediately, just before changes need to be seen